tests/EndToEnd/GetAttribute/narrowing.twig

```diff
@@ -10,4 +10,4 @@
     {% assert_type user.lastPurchaseAt.format('Y-m-d') '*ERROR*' %}
 {% endif %}
 
-{% assert_type user.lastPurchaseAt.format('Y-m-d') '*ERROR*' %}
+{% assert_type user.lastPurchaseAt.format('Y-m-d') 'string' %}
```

Comment:
This is the new behavior I expect (and get with this PR).

See https://phpstan.org/r/94b609fd-bc4e-4f52-919a-57110c075fbe
- An error is reported
- The type is still understood as string